### PR TITLE
fix: Registration failure and other bugs fixed

### PR DESCRIPTION
**Pull request by ZhaoKang0610**
### Description

Fixes # (issue numbers)

### Checklist

- [ ] I have explained the need for this PR and the problem it solves
- [ ] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [ ] I have verified that this change is backward compatible

### CODE REVIEW

File: apps/workbench/backend/db/sql/workbench.sql [hunk 1]
```diff
@@ -84,7 +84,7 @@ CREATE TABLE `workspaces` (
                               `id` char(6) NOT NULL COMMENT '工作空间ID',
```
Comment (yuluo-yx) [inline apps/workbench/backend/db/sql/workbench.sql:84]: Should use english comment？

File: apps/workbench/backend/service/mail/mail.go [hunk 2]
```diff
@@ -4,16 +4,16 @@ import (
 	"bytes"
 	"context"
 	"fmt"
+	"github.com/deamgo/workbench/initialize"
```
Comment (yuluo-yx) [inline apps/workbench/backend/service/mail/mail.go:7]: pkg order

File: apps/workbench/docker/.gitignore [hunk 3]
```diff
@@ -1,2 +1,3 @@
 ./apps/bakcend
-./nginx/html
\ No newline at end of file
+./nginx/html
```
Comment (yuluo-yx) [inline apps/workbench/docker/.gitignore:2]: Should move ignore item to root dir？

File: apps/workbench/backend/db/sql/workbench.sql (hunk 1 above)
Comment (ZhaoKang0610) [reply apps/workbench/backend/db/sql/workbench.sql:84]: I have changed the description to English

File: apps/workbench/backend/service/mail/mail.go (hunk 2 above)
Comment (ZhaoKang0610) [reply apps/workbench/backend/service/mail/mail.go:7]: okey

**Review by iLeepop — APPROVED** [review]
LGTM.

**Pull request merged**